### PR TITLE
Mobile monitoring templates

### PR DESCRIPTION
**Pull request by wei-lee**
## Verification Steps

If you want to run the playbook:

* checkout the branch
* run the following commands against an existing Integreatly cluster:
```
ansible-playbook -i inventories/hosts playbooks/upgrades/upgrade.yaml -e mobile_security_service=true -e eval_mobile_security_service_namespace=test-mss
ansible-playbook -i inventories/hosts playbooks/install_middleware_monitoring.yml -e mobile_security_service=true -e eval_mobile_security_service_namespace=test-mss
```
* You may see some errors but as long as they are not related to the mobile security service, then it's ok. 
* When both are finished, login to the Grafana dashboard of the `Managed Service Monitoring` project, and you should see dashboards for both MSS service and its operator.

If you don't want to run the playbook, you can check this cluster where I have already ran the playbook: https://grafana-route-middleware-monitoring.apps.mobile-6cab.openshiftworkshop.com/?orgId=1. Check the dashboards for MSS service and its operator. There won't be any data for the moment as Prometheus itself is not working properly.

- [ ] Tested Installation
- [ ] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

### CODE REVIEW

**Comment by wei-lee** [comment]
@grdryn if you can help review this will be great.

File: roles/mobile_security_service/templates/mss_operator_prometheus_rule.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+          description: "The mobile-security-service-operator has been down for more than 5 minutes. "
+          summary: "The mobile-security-service-operator is down. For more information see on the MSS operator https://github.com/aerogear/mobile-security-service-operator"
+          sop_url: "https://github.com/aerogear/mobile-security-service-operator/blob/0.2.0/SOP/SOP-operator.adoc"
+      - alert: MobileSecurityServicePodCount
+        annotations:
+          description: "The Pod count for the mobile-security-service has changed in the last 5 minutes and is less than the minimal required value."
+          summary: "Pod count for mobile-security-service is {{ '{{' }} printf '%.0f' $value {{ '}}' }}. Expected 3 pods. For more information see on the MSS operator https://github.com/aerogear/mobile-security-service-operator"
+          sop_url: "https://github.com/aerogear/mobile-security-service-operator/blob/0.2.0/SOP/SOP-operator.adoc"
+        expr: |
+          (1-absent(kube_pod_status_ready{condition="true", namespace="mobile-security-service"})) or sum(kube_pod_status_ready{condition="true", namespace="{{ mobile_security_service_namespace }}"}) < 3
```
Comment (grdryn) [inline roles/mobile_security_service/templates/mss_operator_prometheus_rule.yml.j2]:
```suggestion
          (1-absent(kube_pod_status_ready{condition="true", namespace="{{ mobile_security_service_namespace }}"})) or sum(kube_pod_status_ready{condition="true", namespace="{{ mobile_security_service_namespace }}"}) < 3
```

^maybe, I'm not sure

Comment (wei-lee) [reply roles/mobile_security_service/templates/mss_operator_prometheus_rule.yml.j2]: good catch!

**Comment by wei-lee** [comment]
@odra do you mind taking a look and approve the PR if it's ok given that you are one of the code owners?

**Comment by odra** [comment]
:eyes:

**Pull request merged**